openhtf/core/test_record.py

```diff
@@ -48,6 +48,12 @@ class Attachment(collections.namedtuple('Attachment', 'data mimetype')):
   def sha1(self):
     return hashlib.sha1(self.data).hexdigest()
 
+  def _asdict(self):
+    # Don't include the attachment data when converting to dict.
+    return {
```

Comment:
Looks like this isn't a problem with the test, rather I broke the`inline_attachments` feature in json_factory.

I can't find anyone that's using this feature, but I've added a commit to make it work it just in case. I tested it with all_the_things.py.